addons/website_livechat/static/src/scss/website_livechat.edit_mode.scss

```diff
@@ -0,0 +1,5 @@
+body.editor_enable {
+    .o_livechat_button, .o_thread_window {
+        display: none;
+    }
+}
```

Comment:
We have a class for this `css_editable_mode_hidden`, but I guess it's fine like this and easier than having to `t-jquery` xpath 2 static templates to add the class in this case.